beacon/manager.go

```diff
@@ -276,10 +280,14 @@ func (s *Manager) RunPayloadAttributesSubscription(ctx context.Context, state St
 			continue
 		}
 
-		if fork := state.Fork().Version(structs.Slot(proposalSlot)); fork == structs.ForkAltair || fork == structs.ForkBellatrix {
+		if fork := state.Fork().Version(structs.Slot(slot)); fork == structs.ForkAltair || fork == structs.ForkBellatrix {
 			continue
 		}
 
+		if randao := state.Randao(slot); randao.Randao != "" {
```

Comment:
Is this possible at this point? The new checks added above should prevent us from processing the same slot twice, so how would this have already been set?

Reply:
And I don't think it blocks it from being replaced. The code later on set this to the new value.